### PR TITLE
Update readme & docs

### PR DESCRIPTION
**Pull request by voruti**

### CODE REVIEW

**Comment by rhyst** [comment]
I think the python version update is not correct, and I have fixed the README so I will close this PR

**Comment by voruti** [comment]
Can you give more information on why this Python version is supported? Especially with https://github.com/rhyst/idasen-controller/issues/32#issuecomment-1786104138 in mind?

**Comment by rhyst** [comment]
Sorry, you are correct and I did not read your comment carefully. I did intend to support 3.8 so I have removed the union type operator.

File: README.md [hunk 1]
```diff
@@ -173,7 +173,7 @@ poetry install
 Then you can run the script with:
 
 ```
-poetry run idasen_controller/main.py <command>
+poetry run python -m idasen_controller.main <command>
```
Comment (voruti) [inline README.md:176]: > and I have fixed the README

Isn't the `-m` required here? https://github.com/rhyst/linak-controller/blob/master/README.md?plain=1#L178

**Pull request closed without merging**